.github/workflows/test.yml

```diff
@@ -4,35 +4,37 @@ on:
   workflow_dispatch:
     inputs:
       environment:
-        type: string
+        type: environment
         description: Name of the environment to use
+        default: "oee-dev"
+        options:
+          oee-dev: oee-dev Environment
         required: true
       release_body:
         type: string
         description: GitHub Release Message
-        default: "GitHub Release"
```

Comment:
I think we dont need to specify the release body here since the release job should not be triggered from the test workflow...